<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(178, 172, 136);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_dashboard">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 230);</string>
    </property>
    <property name="text">
     <string>Finance Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savings_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(79, 52, 0, 179);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Savings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phone_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(79, 52, 0, 179);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Phone Payment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="expendables_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(79, 52, 0, 179);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Expendables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(79, 52, 0, 179);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="others_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(79, 52, 0, 179);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Others</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
